packages/react-component-library/src/components/PhaseBanner/README.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Phase Banner
+description: A simple banner to indicate the phase of the project.
+---
+
+## Usage
+
+The banner should be used directly underneath the main site header, before the content begins. If used with no props then it will look exactly like the example image below, however it can be customised to with any text that is required.
+
+## Examples
+
+<img src="images/example.gif" width="664" />
```

Comment:
I wasn't sure how we'd copy the images in the gatsby scripts so I always prefixed my images with the component name just in case.

Reply:
good call.